### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by mastak** [bkjones/pyrabbit#45]
How about new release?
Master branch contain a lot if fixes, but last release was in 2014 :)

**Comment by psliwka** [comment]
+1

**Comment by mwicat** [comment]
Yeah, would be good if we could deploy proper way with up to date version in pypi. @bkjones ?

**Comment by rnt** [comment]
@bkjones please! :+1:

**Comment by pg1671** [comment]
+1

**Comment by scottder** [comment]
+1 if there is any hope of this. I'd love to have that verbose option in there.

**Comment by barakalon** [comment]
+1

**Comment by deslum** [comment]
I fork and update module. Welcome https://github.com/deslum/pyrabbit2
